skbio/stats/ordination/tests/test_ordination_results.py

```diff
@@ -75,6 +75,47 @@ def test_str(self):
                                     pd.Series(np.array([4.2])), samples_df))
         self.assertEqual(obs.split('\n'), exp.split('\n'))
 
+    def test_rename(self):
+        rename_dict_samples = {'Site2' : 'B', 'Site1' : 'A', 'Site3' : 'C'}
+        rename_dict_features = {'Species2' : 'Y', 'Species1' : 'X', 'Species3' : 'Z'}
+
+        self.ordination_results.rename('samples', rename_dict_samples)
+        self.assertEqual(list(self.ordination_results.samples.index), ['A', 'B', 'C'])
+
+        self.ordination_results.rename('features', rename_dict_features)
+        self.assertEqual(list(self.ordination_results.features.index), ['X', 'Y', 'Z'])
+
+        rename_dict_samples = {'A' : '001', 'B' : '010'}
+        rename_dict_features = {'X' : 'abc', 'Y' : 'def'}
+
+        self.ordination_results.rename('samples', rename_dict_samples, strict=False)
+        self.assertEqual(list(self.ordination_results.samples.index), ['001', '010', 'C'])
+
+        self.ordination_results.rename('features', rename_dict_features, strict=False)
+        self.assertEqual(list(self.ordination_results.features.index), ['abc', 'def', 'Z'])
+
+        rename_dict_samples = {'001' : 'he', '010' : 'llo', 'C' : 'wor', 'D' : 'ld'}
+        rename_dict_features = {'abc' : 'go', 'def' : 'od', 'Z' : 'by', 'W' : 'e'}
+
+        with self.assertRaises(ValueError):
```

Comment:
From this line on, the order of axis and mapper have been shuffled. Are these intended to test the wrong order, or the wrong ID mappings? If the latter, the code needs to be modified. Also, if you want to be more explicit about the error that is raised, you can do `self.assertRaises(ValueError, msg="The ids in mapper are different from the ids in self.samples.")`.

Reply:
Good catch. Above this though it's also axis then mapper but the function signature is mapper then axis?

Reply:
Thanks for pointing this out! It should be taken care of now.